.devcontainer/docker-compose.yml

```
@@ -15,6 +15,10 @@ services:
       - ${HOME}${USERPROFILE}/.gitconfig:/home/opam/.gitconfig
       - ${HOME}${USERPROFILE}/.gitignore_global:/home/opam/.gitignore_global
       - /var/run/docker.sock:/var/run/docker.sock
+      # Add database folders to dev container
+      - db-root:/dbs/db-root/:ro
+      - db-tenant:/dbs/db-tenant/:ro
```

Please use full name 
```suggestion
      - db-root:/databases/db-root/:ro
      - db-tenant:/databases/db-tenant/:ro
```